### PR TITLE
✨ [amp-render] Hide placeholder on `refresh` action and other minor changes

### PR DESCRIPTION
**Pull request by dmanek**
Partial for #32477.

Few changes in this PR:
1. Hide the placeholder on `refresh` action (similar to `amp-list`)
2. Remove check for `amp-render` experiment flag 
3. Minor doc update

### CODE REVIEW

File: extensions/amp-render/1.0/amp-render.js [hunk 1]
```diff
@@ -225,7 +215,7 @@ export class AmpRender extends BaseElement {
         this.togglePlaceholder(false);
       },
       'onRefresh': () => {
-        this.togglePlaceholder(true);
+        this.togglePlaceholder(false);
         this.toggleFallback(false);
       },
```
Comment (caroqliu) [inline extensions/amp-render/1.0/amp-render.js:220]: Won't these already be toggled off on refresh? Also, would we want to toggle the loading on refresh, possibly in the Preact layer?

File: extensions/amp-render/1.0/amp-render.js [hunk 2]
```diff
@@ -116,15 +115,6 @@ export class AmpRender extends BaseElement {
     this.src_ = null;
   }
 
-  /** @override */
-  isLayoutSupported(layout) {
-    userAssert(
-      isExperimentOn(this.win, 'amp-render'),
-      'Experiment "amp-render" is not turned on.'
-    );
```
Comment (caroqliu) [inline extensions/amp-render/1.0/amp-render.js:124]: Is this removal intended? Since there are validations, removing this experiment assertion will affectively launch the component.

File: extensions/amp-render/1.0/amp-render.js (hunk 1 above)
Comment (dmanek) [reply extensions/amp-render/1.0/amp-render.js]: 1. That's right, these will already be toggled off in `onReady` callback, therefore removed `onRefresh`. 
2. Logic to show/hide placeholder & fallback happens on the AMP side, so moving it to the Preact layer will not be trivial

File: extensions/amp-render/1.0/amp-render.js (hunk 2 above)
Comment (dmanek) [reply extensions/amp-render/1.0/amp-render.js:124]: Yes, this is intended. Based on conversation with @nainar & @alanorozco, components launched behind an experiment typically don't get high usage, while we want this to get usage and collect feedback.

File: extensions/amp-render/1.0/amp-render.js (hunk 1 above)
Comment (caroqliu) [reply extensions/amp-render/1.0/amp-render.js]: 2. No, we should not move placeholder/fallback logic to Preact layer. I am more wondering if Preact layer `refresh` callback should also call `onLoading`? (In AMP layer, this would mean retoggling the loading spinner, which I can't recall if 0.1 amp-list does or not, but seems like a reasonable UX behavior)

**Review by caroqliu — APPROVED** [review]

File: extensions/amp-render/1.0/amp-render.js (hunk 1 above)
Comment (dmanek) [reply extensions/amp-render/1.0/amp-render.js]: I see what you mean. By default amp-list does not show the loading indicator during refresh, unless `reset-on-refresh` attribute is specified. Currently, we don't support `reset-on-refresh` for amp-render, but it's something we can revisit if we get some feedback on it.

Comment (caroqliu) [reply extensions/amp-render/1.0/amp-render.js]: Sounds good, thank you!
